//	W.A.J. P to implement the above program (pro.no-B27) using nesting of try-catch
//block. try {
//try
//{//code}
//catch (Exception e)
//{//code}
//catch (Exception e)
//{//code

package com.assignment;

public class Assignment37
{
	public static void main(String[] args)
	{
		try 
		{
			
		} catch (ArithmeticException e) 
		{
			// TODO: handle exception
		} catch (ArrayIndexOutOfBoundsException e)
		{
			
		} catch (Exception e)
		{
			
		}
	}
}
